manifests/selinux.pp

```diff
@@ -20,29 +20,13 @@
       persistent => true,
       value      => $_sebool_export_all_ro
     }
-    if ( $facts['os']['release']['major'] < '7' ){
-      selboolean { 'allow_rsync_anon_write':
-        persistent => true,
-        value      => $_sebool_anon_write
-      }
-      selboolean { 'rsync_use_cifs':
-        persistent => true,
-        value      => $_sebool_use_cifs
-      }
-      selboolean { 'rsync_use_nfs':
-        persistent => true,
-        value      => $_sebool_use_nfs
-      }
+    selboolean { 'rsync_anon_write':
```

Comment:
Need to remove `$_sebool_use_nfs` and `$_sebool_cifs`, as they are no longer used.